yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -959,7 +959,7 @@ func convertAnalyzeSchemaIssueToAssessmentIssue(analyzeSchemaIssue utils.Analyze
 
 		// Reason in analyze is equivalent to Description of IssueInstance or AssessmentIssue
 		// and we don't use any Suggestion field in AssessmentIssue. Combination of Description + DocsLink should be enough
-		Description: analyzeSchemaIssue.Reason + " " + analyzeSchemaIssue.Suggestion,
+		Description: lo.Ternary(analyzeSchemaIssue.Suggestion == "", analyzeSchemaIssue.Reason, analyzeSchemaIssue.Reason+" "+analyzeSchemaIssue.Suggestion),
```

Comment:
you can just concat in all cases, right?

Reply:
if suggestion string is empty it was resulting in a extra(trailing) space at the end.